packages/account/src/Sections/Verification/ProofOfAddress/__tests__/proof-of-address-container.spec.tsx

```diff
@@ -0,0 +1,201 @@
+import React from 'react';
+import merge from 'lodash.merge';
+import { screen, render } from '@testing-library/react';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import ProofOfAddressContainer from '../proof-of-address-container';
+import { BrowserRouter } from 'react-router-dom';
+import { useKycAuthStatus } from '../../../../hooks';
+import { AUTH_STATUS_CODES } from '@deriv/shared';
+import userEvent from '@testing-library/user-event';
+
+type TKYCAuthStatusResponse = Partial<ReturnType<typeof useKycAuthStatus>>;
+
+const mock_kyc_auth_status_response: TKYCAuthStatusResponse = {
+    kyc_auth_status: {
+        address: { status: 'none' },
+        identity: { status: 'none' },
+    },
+    isLoading: false,
+    isSuccess: false,
+    reFetchKycAuthStatus: jest.fn(),
+};
+
+jest.mock('../../../../hooks', () => ({
+    useKycAuthStatus: jest.fn(() => mock_kyc_auth_status_response),
+}));
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    getPlatformRedirect: jest.fn(),
+    WS: {
+        wait: jest.fn().mockResolvedValue({}),
+    },
+}));
+
+jest.mock('@deriv/hooks', () => ({
+    ...jest.requireActual('@deriv/hooks'),
+    useFileUploader: jest.fn(() => ({
+        upload: jest.fn(),
+    })),
+}));
+
+jest.mock('Components/poa/common-mistakes/common-mistake-examples', () =>
+    jest.fn(() => <div>MockCommonMistakeExampleComponent</div>)
+);
+
+describe('ProofOfAddressContainer', () => {
+    const store = mockStore({});
+
+    const mergeResponse = (override: TKYCAuthStatusResponse) => merge(mock_kyc_auth_status_response, override);
```

Comment:
let's try to find the way not use lodash, specially in tests.
would be better to extend our deriv utils

Reply:
Do we have deriv util functions that do the same thing?